src/BottomNavigation/BottomNavigation.spec.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import BottomNavigation from './BottomNavigation';
+import BottomNavigationItem from './BottomNavigationItem';
+import getMuiTheme from '../styles/getMuiTheme';
+
+describe('<BottomNavigation />', () => {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+
+  it('determines which BottomNavigationItem is selected', () => {
+    const wrapper = shallowWithContext(
+      <BottomNavigation selectedIndex={0}>
+        <BottomNavigationItem />
+        <BottomNavigationItem />
+        <BottomNavigationItem />
+      </BottomNavigation>
+    );
+    assert.ok(wrapper.childAt(0).node.props.selected, 'index 0 should be selected');
+    assert.notOk(wrapper.childAt(1).node.props.selected, 'index 1 should not be selected');
+    assert.notOk(wrapper.childAt(2).node.props.selected, 'index 2 should not be selected');
```

Comment:
I thing that it would be better to use `assert.strictEqual`.
Why? The smaller to API surface we use the simpler and the better.